public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```

Comment:
Does this prevent graphiql from being indexed by search engines? I'm not sure if we want to prevent it or not.

Reply:
Both of these files were created when I initalized a new app with create react app in order to migrate to the new version. I removed this file now